client/src/map/shareLocationModal.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import styled from 'styled-components';
+import Clipboard from 'react-clipboard.js';
+import { GoClippy, GoBroadcast } from 'react-icons/go';
+import { FaBroadcastTower } from 'react-icons/fa';
+import Modal, { ModalHeader, ModalParagraph } from '../common/modal';
+
+/**
+ * TODO: This URL should probably be an environment variable
+ */
+const baseUrl = 'http://localhost:8080';
+
+const FlexContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const IconColumn = styled.div`
```

Comment:
Are IconColumn and ContentColumn meant to have styles later on? I think it would be good to just use <div> in the jsx, instead of making empty styled component.